workflow-manager/main.go

```diff
@@ -174,9 +170,18 @@ func main() {
 	log.Printf("starting %s version %s - %s. Args: %s", os.Args[0], BuildID, BuildTime, os.Args[1:])
 	flag.Parse()
 
-	ingestorService, ingestorInputBucket := checkStoragePath("--ingestor-input", *ingestorInput)
-	ownValidationService, ownValidationInputBucket := checkStoragePath("--own-validation-input", *ownValidationInput)
-	peerValidationService, peerValidationInputBucket := checkStoragePath("--peer-validation-input", *peerValidationInput)
```

Comment:
I believe these were the only usages of the `checkStoragePath` function, which can now be deleted.